src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md

```diff
@@ -7,15 +7,15 @@ description: |
   PageSpeed Insights (PSI) is a tool for web developers to understand what a
   page's performance is and how to improve it. In this guide, learn how to use
   PSI to extract insights from CrUX and better understand the user experience.
-date: 2018-11-05
+date: 2020-05-28
 ---
 
 [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/)
 (PSI) is a tool for web developers to understand what a page's performance is
 and how to improve it. It uses [Lighthouse](https://developers.google.com/web/tools/lighthouse/)
-to audit a given page and identify opportunities to improve performance. It also
-integrates with the Chrome UX Report ([CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/))
-to show how real users experience performance on the page and the origin in
+to audit the page and identify opportunities to improve performance. It also
+integrates with the [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report/) (CrUX)
+to show how real users experience the page and the origin in
```

Comment:
```suggestion
to show how real users experience the page and the [origin](/same-site-same-origin/#origin) in
```

Reply:
Hmm none of the origins in CrUX specify a port and this link might lead some readers to think that they need to include one when looking up an origin.